core/templates/dev/head/pages/exploration_editor/post_publish_modal_directive.html

```diff
@@ -47,6 +61,11 @@ <h3>Awesome!</h3>
     <p>
       Now that you're a published Oppia teacher, you can share your creation.
     </p>
+    <div class="oppia-share-publish-link">
+      Here's your exploration link :
```

Comment:
There shouldn't be a space before the comma.

Reply:
Done.